Dockerfile

```diff
@@ -20,11 +21,13 @@ RUN cmake CMakeLists.txt && make && cp ./*.a /usr/lib
 
 ADD . /opt/OpenDDS
 
+ARG ACE_CONFIG_OPTION="--doc-group"
+ARG MPC_ROOT="/opt/OpenDDS/ACE_wrappers/MPC"
```

Comment:
This shouldn't be an `ARG` since the configure script knows it already.  After the configure script run, source the setenv.sh script that it generated.

Reply:
It's an ARG so I can change it in the docker build for the OpenDDS/ROS2 version. As far as I know, there's no other way to influence the docker build externally.

Reply:
You don't need to do that for `MPC_ROOT`, which is what this comment is about.

Reply:
The copy has a source directory of `/opt/OpenDDS/ACE_wrappers/MPC` for the `--doc-group option` and `/opt/OpenDDS/ACE_TAO/MPC` for the `--ace-github-latest` option. Are you saying sourceing `setenv.sh` with `--ace-github-latest` would remove the requirement for the copy?

Reply:
All it does is tell you where the MPC repository code is located.  It does this by setting the environment variable `MPC_ROOT`.  This is exactly the source directory you're looking for.

Reply:
Got it. That makes more sense.